core/src/main/scala/chisel3/internal/SourceInfo.scala

```diff
@@ -58,4 +58,4 @@ object SourceInfoMacro {
 
 object SourceInfo {
   implicit def materialize: SourceInfo = macro SourceInfoMacro.generate_source_info
-}
+}
```

Comment:
Interesting implementation choice here. Seems a bit counter-intuitive, but I like the brashness if your approach. Perhaps next time, you can consider alternatives? For now though, I'll let this one change slide.